hbase-client/src/main/java/org/apache/hadoop/hbase/client/Get.java

```diff
@@ -95,6 +95,7 @@ public Get(Get get) {
     this.setFilter(get.getFilter());
     this.setReplicaId(get.getReplicaId());
     this.setConsistency(get.getConsistency());
+    this.setResultMetricsEnabled(get.isResultMetricsEnabled());
```

Comment:
Nit; Thoughts on the name `QueryMetrics`? I think that `ResultMetrics` could be interpreted as "metrics about the result" instead of "metrics about the query operation"

Reply:
Query metrics makes sense, I mostly just wanted a way to distinguish between the existing scan metrics, and these new metrics